setup.py

```diff
@@ -1,26 +1,28 @@
-import setuptools
+#!/usr/bin/env python
+#
+# This file is part of ts_IntegrationTests.
```

Comment:
`ts_IntegrationTests` -> `ts_SalMultiLanguageTests`

Please check the other Python files as well.

Reply:
Fixed